src/Filters/ConvertPsuedoHeadings.php

```diff
@@ -33,7 +33,13 @@ public function apply(string $html): string
         );
 
         $html = preg_replace(
-            '/(<p>(?:\s+)?<strong>(.{0,80}?)<\/strong>(<br *\/?>)?(?:\s+)?<\/p>)+/si',
+            '/(<strong>)\s*?(<\/strong>)/im',
+            ' ',
+            str_ireplace('\n', ' ', $html)
```

Comment:
Should we replace line breaks in a different filter? Might fit more in `RemoveSpaces`

Reply:
Yeah not sure we need this still, I think it is a remnant of something I was trying while in dev.

Reply:
Removed and tests still passing.